sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/platform/Notifications.kt

```diff
@@ -1,9 +1,9 @@
 package co.touchlab.sessionize.platform
 
-expect fun createLocalNotification(title:String, message:String, timeInMS:Long, notificationId: Int)
+expect fun createLocalNotificationOnPlatform(title:String, message:String, timeInMS:Long, notificationId: Int)
```

Comment:
We seem to be missing the implementation of this on iOS

Reply:
For createLocalNotificationOnPlatform? It should be there.